source/Esf.Website/App/shared/layout/horizontalSplitView/horizontalSplitView.component.ts

```diff
@@ -0,0 +1,68 @@
+﻿import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+    selector: 'horizontal-split-view',
+    styleUrls: ['/App/shared/layout/horizontalSplitView/horizontalSplitView.styles.css'],
+    templateUrl: '/App/shared/layout/horizontalSplitView/horizontalSplitView.component.html',
+})
+export class HorizontalSplitViewComponent {
+    @Output() onResize: EventEmitter<any> = new EventEmitter();
+
+    private view1Height: string;
+    private view2Height: string;
+
+    private isDragging: boolean;
+
+    private containerNode: Node;
+
+    constructor() {
+        this.view1Height = "50%";
+        this.view2Height = "50%"
+        this.isDragging = false;
+    }
+
+    private onMouseDown(event: MouseEvent): void {
+        this.startDragging(event);
+    }
+
+    private onMouseMove(event: MouseEvent): void {
+        if (this.isDragging) {
+            this.recalculateHeigths(event);
+        }
+    }
+
+    private onMouseUp(event: MouseEvent): void {
+        this.stopDragging(event);
+    }
+
+    private onMouseLeave(event: MouseEvent): void {
+        this.stopDragging(event);
+    }
+    
+    private startDragging(event: MouseEvent): void {
+        this.isDragging = true;
+        this.containerNode = event.srcElement.parentNode;
```

Comment:
event.srcElement works only is available only in chrome for this event.
To fix it we can use event.currentTarget.parent or elementRef.nativeElement.parentNode where elementRef is injected from angular (the type comes from import {ElementRef } from '@angular/core')